pkg/common/parameters.go

```diff
@@ -208,6 +218,15 @@ func ExtractAndDefaultParameters(parameters map[string]string, driverName string
 				return p, fmt.Errorf("parameters contain invalid value for %s parameter: %w", ParameterKeyStoragePools, err)
 			}
 			p.StoragePools = storagePools
+		case ParameterKeyResourceTags:
+			paramResourceTags, err := ConvertTagsStringToMap(v)
+			if err != nil {
+				return p, fmt.Errorf("parameters contain invalid tags parameter: %w", err)
```

Comment:
Can you specify the exact name of the parameter key in this error? This will make it easier for the user to address the error (by knowing exactly which parameter key to modify).

Reply:
@pwschuurman I have added the full string containing the parent ID, tag key and value in the error message returned by `ConvertTagsStringToMap()`.

Reply:
I'm looking for the actual key that the user would be familiar with (eg: ParameterKeyResourceTags) to be present in the output error message. This will allow the user to know what parameter to correct in their gRPC call. The end user will likely be a Kubernetes user, and this parameter should map to a parameter key they are familiar with, so they can update the corresponding parameter on their StorageClass. Without that context, they'll just be left with the value, and it won't be as clear about what parameter to change to fix the error they're seeing.

Reply:
Got it. Added the name of the parameter key in the error message.